### DISCUSSION TITLE
Conductor UI - Search with workflowType , Status not returning any results.

### DISCUSSION THREAD

**Discussion by sukruthiarjula** [Netflix/conductor#2871] [Q&A]
We are upgrading to v3.6.0 with ES7 , build is successful and while testing in Conductor UI search with workflowType and Status not returning any result. Search with workflowId is working fine. May I know what would be the reason for this??

**Comment by apanicker-nflx** [comment]
@erikbrgr Can you share your experience working with ES7 and if you have run into a similar issue?

**Comment by erikbrgr** [comment]
In Conductor 3.5.1, I cannot reproduce the issue.

@sukruthiarjula, if possible, can you use the code in this article https://betterprogramming.pub/getting-started-with-netflix-conductor-in-net-using-supersimpleconductor-ed8a02120c1 to set up a clean Conductor 3.5.1 environment, and see if the issue persists?

If I have time, I can set up a 3.6.0 environment and recheck, but that might be a while...

**Comment by erikbrgr** [comment]
@sukruthiarjula Also, maybe explore the es7 indices to see if the issue may be with missing data. For example, executing the following from a terminal window will list all workflows:
`curl -X GET "localhost:9200/conductor_workflow/_search?pretty"`
Obviously, you'll need to replace the url with yours.

**Reply by sukruthiarjula** [reply]
> @sukruthiarjula Also, maybe explore the es7 indices to see if the issue may be with missing data. For example, executing the following from a terminal window will list all workflows: `curl -X GET "localhost:9200/conductor_workflow/_search?pretty"` Obviously, you'll need to replace the url with yours.

@erikbrgr Yes I am able to get list of workflows with curl -X GET "localhost:9200/conductor_workflow/_search?pretty"
